esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -398,10 +339,7 @@ void CWsESDLConfigEx::ensureESDLServiceBindingRegistry(const char * espProcName,
     if (!espBindingName || !*espBindingName)
         throw MakeStringException(-1, "Unable to ensure ESDL service binding registry in dali, esp binding name not available");
 
-    if (!ensureSDSPath(ESDL_BINDINGS_ROOT_PATH))
-        throw MakeStringException(-1, "Unable to connect to ESDL Service binding information in dali %s", ESDL_BINDINGS_ROOT_PATH);
-
-    Owned<IRemoteConnection> globalLock = querySDS().connect(ESDL_BINDINGS_ROOT_PATH, myProcessSession(), RTM_LOCK_WRITE, SDS_LOCK_TIMEOUT_DESDL);
+    Owned<IRemoteConnection> globalLock = querySDS().connect(ESDL_BINDINGS_ROOT_PATH, myProcessSession(), RTM_LOCK_WRITE | RTM_CREATE_QUERY, SDS_LOCK_TIMEOUT_DESDL);
```

Comment:
Is this and similar elsewhere necessary any more? As you have:
```
    if(!ensureSDSPath(ESDL_BINDINGS_ROOT_PATH))
        throw MakeStringException(-1, "Could not ensure '%s' element on Dali", ESDL_BINDINGS_ROOT_PATH);
```
.. in CWsESDLConfigEx::init()